sympy/polys/tests/test_rootoftools.py

```diff
@@ -690,8 +689,8 @@ def test_issue_15920():
 def test_issue_19113():
     eq = y**3 - y + 1
     # generator is a canonical x in RootOf
-    assert str(Poly(eq).real_roots()) == '[CRootOf(x**3 - x + 1, 0)]'
+    assert str(Poly(eq).real_roots()) == '[CRootOf(_x**3 - _x + 1, 0)]'
```

Comment:
Does `dummy_eq` work now?

Reply:
I need to add some tests for what is changed here but `dummy_eq` always worked for `RootOf`. The purpose of this particular test though is to make sure that the symbols is `x` regardless of what `__eq__` or `dummy_eq` do.